.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,9 @@
 - [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
 - [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
-- [ ] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
-- [ ] Have you tested your manifest locally with `winget install -m <manifest>`?
+- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
+- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
+- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
+
+Note: `<path>` is the name of the directory containing the manifest you're submitting.
```

Comment:
Maybe `name of the file or directory` to account for both singleton and split manifests.